api/application.js

```diff
@@ -297,6 +297,8 @@ export async function setSystemMenu (o) {
       const binding = match[3]
       if (label.length === 0) {
         err = 'Missing label'
+      } else if (label.includes(':')) {
+        err = 'Invalid label contains ":"'
```

Comment:
I think it'd be preferable to allow labels with `:` in the future by changing the native code to search for the last `:` instead of splitting by all `:` characters.

For now, an error is thrown.